### ISSUE TITLE
Override config file arguments from CLI

### ISSUE THREAD

**Issue by ghenadiibatalski** [lwsjs/local-web-server#176]
Hello and thank you for this wonderful product! I'm facing a problem, that i need to override at least the `--directory` from a command line even if this is already configured via the config file. Could you please make a CLI arguments a higher precedence over the config file?

Thanks and kind regards, Gena

**Comment by 75lb** [comment]
Hi, yes this looks like a bug.. if you set the `--config` flag you can inspect the config that `ws` is using, e.g.

```
$ ws --port 8030 --config
{
  port: 8030,
  stack: MiddlewareStack(3) [ LwsAuth, Static ],
  view: CliView {}
}
```

Do you find that the `--config` output is correct but is not being respected? I imagine this is a quick fix, will look into it..

**Comment by 75lb** [comment]
Fixed and released in [local-web-server v5.4.0](https://github.com/lwsjs/local-web-server/releases/tag/v5.4.0)
